### ISSUE TITLE
Not working on `Ubuntu 20.04`

### ISSUE THREAD

**Issue by shubhexists** [OthersideAI/self-operating-computer#39]
I tried running this on Ubuntu 20.04, it is getting the commands right in the terminal, but it is not actually navigating to the desired location. For example, I asked it to open brave and navigate to this repository... It is getting the steps right, but it could not open Brave....

# Possible Issue
- I think the method to search apps is different in MacOS ( I haven't used Mac, so I am not aware though) and in my system....
Hence, it might mot be able to go to the apps menu to actually search for any software.. Maybe?

**Comment by michaelhhogue** [comment]
@shubhexists This is a known issue. For now, the search command is tailored for Mac OS. Restricting your prompt to reference UI elements which are already on-screen is a work-around. Having a set way to open a launcher/menu via key shortcuts is going to be more difficult in Linux as there isn't one set way to open the launcher/menu like in Mac OS.

One possible way to resolve this issue on GNU/Linux could be to simply prompt the user for their keyboard shortcut to open a launcher/menu/search.

**Comment by stefangrotz** [comment]
Windows Key + Type should work in most linux distros (+ in Windows). I will test it later and maybe also create a PR with a fix for Ubuntu.

**Comment by shubhexists** [comment]
Should I try adding Linux here 
https://github.com/OthersideAI/self-operating-computer/blob/bc3c5fdbac7ad74cbcada0e55963fd7d2004de6e/operate/main.py#L546
and see what the result is ?

**Comment by michaelhhogue** [comment]
@shubhexists Yeah this is a good default. It personally isn't the key I use to open the menu (I have a keyboard intended for Macs, so I have command/alt open the menu), so I'm probably biased against it. But you are right, this is certainly most common for Linux.

**Comment by shubhexists** [comment]
Hi I solved it and now it is searching applications using the windows key...   
If you agree, I can make a PR adding this to the main repository?

# Demo
https://github.com/OthersideAI/self-operating-computer/assets/110319892/598f0753-5fc0-475e-8e09-82bd0f71d93c

**Comment by michaelhhogue** [comment]
That looks great! This should be a good PR.

**Comment by joshbickett** [comment]
Merged the PR. Thanks for monitoring @michaelhhogue and adding the code @shubhexists
